.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,44 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.245.0/containers/typescript-node
+{
+	"name": "CipherEditor Dev Container Suite",
+	"build": {
+		"dockerfile": "Dockerfile",
+		// Update 'VARIANT' to pick a Node version: 18, 16, 14.
+		// Append -bullseye or -buster to pin to an OS version.
+		// Use -bullseye variants on local on arm64/Apple Silicon.
+		"args": { 
+			"VARIANT": "16-bullseye"
+		}
+	},
```

Comment:
The bullseye variant only work for local arm64/Apple Silicon, if I understand this correctly. What should users on other platforms do?

Reply:
No? It's all containerization software, so it will always run wherever a docker installation can run. That's the beauty of containerization software

Reply:
Hmm, I'll try spinning it on my desktop which is amd64 tomorrow, I think what they mean was that the buster variants of Debian were not compiled for arm64. Although I can definitely assure you that popular images like these are compiled for all architectures

Reply:
![image](https://user-images.githubusercontent.com/35210888/187401082-b58a723d-62b2-4510-b4e8-c2a137216764.png)
Perfectly works on my mac (Apple Silicion) and my desktop (x86)

Reply:
Good work! ✨